CMakePresets.json

```diff
@@ -91,5 +91,30 @@
         "rhs": "Windows"
       }
     }
+  ],
```

Comment:
These last second additions really drag out the review process, because now I'll have to check what this is, check that it works and determine if it's worth it. You can leave this as it seems to work, but just keep in mind that if it's not essential then we can iterate on this in a followup PR.

Reply:
I'm aware of that, but this one is a pretty useful addition.